slides/k8s/staticpods.md

```diff
@@ -0,0 +1,239 @@
+# Static pods
+
+- Pods are usually created indirectly, through another resource:
+
+  Deployment, Daemon Set, Job, Stateful Set ...
+
+- They can also be created directly
+
+- This can be done by writing YAML and using `kubectl apply` or `kubectl create`
+
+- Some resources (not all of them) can be created with `kubectl run`
+
+- Creating a resource with `kubectl` requires the API to be up
+
+- If we want to run the API server (and its dependencies) on Kubernetes itself ...
+
+  ... how can we create API pods (and other resources) when the API is not up yet?
+
+---
+
+## In theory
+
+- Each component of the control plane can be replicated
+
+- We could set up the control plane outside of the cluster
+
+- Then, once the cluster is up, create replicas running on the cluster
```

Comment:
What do we mean by "up"? Launched? Responding on an API endpoint? Processing work in some other measurable way?

Reply:
I suggest "fully operational". Would that be OK?

A formal definition would be: "API server is available, which means an etcd cluster has to be functional, and cluster components like the controller manager and the scheduler have to work as well. And we need at least one workable node."